File: src/main/java/com/uwetrottmann/trakt5/entities/Season.java
package com.uwetrottmann.trakt5.entities;

import org.threeten.bp.OffsetDateTime;

import java.util.List;

public class Season {

    public Integer number;
    public SeasonIds ids;

    public String title;
    public String overview;
    public String network;
    public OffsetDateTime first_aired;
    public Double rating;
    public Integer votes;
    public Integer episode_count;
    public Integer aired_episodes;
    public List<Episode> episodes;
}

File: src/main/kotlin/com/redissi/trakt/entities/Season.kt
package com.redissi.trakt.entities

import com.squareup.moshi.Json
import java.time.OffsetDateTime

class Season(
    var number: Int? = null,
    var ids: SeasonIds? = null,
    var title: String? = null,
    var overview: String? = null,
    var network: String? = null,
    @Json(name = "first_aired")
    var firstAired: OffsetDateTime? = null,
    var rating: Double? = null,
    var votes: Int? = null,
    @Json(name = "episode_count")
    var episodeCount: Int? = null,
    @Json(name = "aired_episodes")
    var airedEpisodes: Int? = null,
    var episodes: List<Episode>? = null
)